app/assets/javascripts/application.js

```diff
@@ -7,6 +7,8 @@
 //= require health_check
 //= require highlight
 //= require clipboard
+//= require d3.v3
```

Comment:
We can also load them from rails-assets.org :-) That way we don't need to keep the code in our own repo, and can update it just like a gem :-D